source/fab/steps/analyse.py

```diff
@@ -437,3 +431,22 @@ def _add_unreferenced_deps(self, symbols: Dict[str, Path],
             # add the file and it's file deps
             sub_tree = extract_sub_tree(source_tree=all_analysed_files, root=analysed_fpath)
             build_tree.update(sub_tree)
+
+
+def _load_analysis_file(fpath) -> Dict[Path, AnalysedFile]:
```

Comment:
This function, on the other hand, does have an `fpath` argument. Have they got mixed up? Also, does `fpath` have a type?

Reply:
Yes, they were one function, now split. Thanks.